### ISSUE TITLE
Incomplete declarations on some successful tasks

### ISSUE THREAD

**Issue by mstokes-f5** [F5Networks/f5-appsvcs-extension#457]
### Environment
 * Application Services Version: 3.27.0
 * BIG-IP Version: 15.1.2.1

### Summary
Sometimes a task returned by the `/task` endpoint does not have the full submitted declaration even though the task was successful. Previous versions of AS3 would always return the submitted declarations on success.

### Steps To Reproduce
This appears intermittent to me. Sometimes my submitted declaration is in the task object and sometimes it is not. If/when someone has time to look at this, I can work with them on repro (including granting access to a box demonstrating the behavior). On my machine, I can reliably submit this (messy, not pared-down) declaration to reproduce the problem:
```json
{                                                                                                                                   
  "Common": {                                                      
    "class": "Tenant",                                                                                                                      
    "Shared": {                                                                                                                             
      "class": "Application",                
      "template": "shared"
    }              
  },                                  
  "class": "ADC",
  "schemaVersion": "3.0.0",
  "controls": {                                                                                                                             
    "class": "Controls",                                                                                                                    
    "userAgent": "f5-appsvcs-templates/1.9.0-rc.5"                                                                                          
  },                                                                                                                                        
  "id": "fast%create%foo%bar%1",             
  "updateMode": "selective",
  "foo": {        
    "class": "Tenant",         
    "bar": {   
      "class": "Application",
      "template": "udp",                                                                                                                    
      "serviceMain": {                                                                                                                      
        "class": "Service_UDP",                                                                                                             
        "virtualAddresses": [                                                                                                               
          "192.0.2.1"                                 
        ],
        "virtualPort": 5555,
        "pool": "bar_Pool1"    
      },       
      "bar_Pool1": {
        "class": "Pool",                                 
        "monitors": [       
          "icmp"
        ],
        "members": [
          {
            "serverAddresses": [
              "192.0.2.2"
            ],
            "servicePort": 5555
          }
        ]
      },
      "constants": {
        "class": "Constants",
        "fast": {
          "template": "examples/simple_udp_defaults",
          "setHash": "aebbea8fb59b96168e1857fc9bdf411d4b26d2636d828ac486b4d74d2c4f9688",
          "view": {
            "tenant_name": "foo",
            "application_name": "bar",
            "virtual_address": "192.0.2.1",
            "virtual_port": 5555,
            "server_addresses": [
              "192.0.2.2"
            ],
            "service_port": 5555
          },
          "lastModified": "2021-05-13T04:17:45.951Z",
          "ipamAddrs": {}
        }
      }
    }
  }
}
```

### Expected Behavior
Ideally, the submitted declaration is always available in the task, but going back to declarations being consistently available in successful tasks would be sufficient.

### Actual Behavior
I get something like this:
```json
   {
      "id": "d0248c0d-171e-4559-8651-04f65e3296e7",
      "results": [
        {
          "code": 200,
          "message": "success",
          "lineCount": 24,
          "host": "localhost",
          "tenant": "foo",
          "runTime": 1026
        }
      ],
      "declaration": {
        "class": "ADC",
        "schemaVersion": "3.0.0",
        "controls": {
          "class": "Controls",
          "userAgent": "f5-appsvcs-templates/1.9.0-rc.5",
          "archiveTimestamp": "2021-05-13T04:17:28.492Z"
        },
        "id": "fast%create%foo%bar%1",
        "updateMode": "selective"
      }
    }
```

**Comment by dstokesf5** [comment]
Thank you for reporting this. It has been added to our internal product backlog as AUTOTOOL-2499.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 3.29.0 release.
